custom_components/tuya_local/devices/ustellar_ut99911_rgb.yaml

```diff
@@ -0,0 +1,174 @@
+name: Smart Monitor Light Bar
```

Comment:
HA naming convention is to only capitalise the first word of device and entity names.  Also, this will be prepended to every entity id, so it is best to keep it short by dropping redundant words like "Smart", "WiFi" etc from the actual product name.

Reply:
Renamed, thank you.